src/Process/Pool/Logger/Message/SerializableProcess/FromProcessModel/Builder.yml

```diff
@@ -0,0 +1,7 @@
+services:
+  neighborhoods.kojo.process.pool.logger.message.serializable_process.from_process_model.builder:
+    class: Neighborhoods\Kojo\Process\Pool\Logger\Message\SerializableProcess\FromProcessModel\Builder
+    calls:
+      - [setProcessPoolLoggerMessageProcessFromProcessModelFactory, ['@neighborhoods.kojo.process.pool.logger.message.serializable_process.from_process_model.factory']]
```

Comment:
The alias (process.pool.logger.message.**serializable_process**.from_process_model.factory) doesn't match the setter (Process\Pool\Logger\Message\\**Process**\FromProcessModel\Factory

Is that intended and does it work?

I suspect that the `AwareTrait`s weren't re-generated after you changed `Process` to `SerializableProcess` in the filesystem/classes